build.gradle

```diff
@@ -24,7 +24,7 @@ apply plugin: 'com.github.sherter.google-java-format'
 apply plugin: 'io.codearte.nexus-staging'
 
 // TODO: Populate this from dependencies.properties version property (for proper Gradle-Bazel sync)
-project.version = "1.57.1" // {x-version-update:gax:current}
+project.version = "1.57.2" // {x-version-update:gax:current}
```

Comment:
@chingor13 not sure if I should upgrade a version. let me know what you think. Thanks